### DISCUSSION TITLE
Unable to install @rhds/elements

### DISCUSSION THREAD

**Discussion by eyevana** [RedHat-UX/red-hat-design-system#378] [Q&A] [answered]
Hey, all! I'm one of the devs on the Red Hat Ecosystem Catalog team and I'm having trouble installing the package for <rh-footer>. I'm using node 16 but I see the following error when trying to install it with `npm install @rhds/elements`: 

npm ERR! code 127
npm ERR! path /Users/ivrodrig/Documents/projects/ecosystem-catalog/node_modules/@rhds/elements
npm ERR! command failed
npm ERR! command sh -c run-s postinstall:**
npm ERR! sh: run-s: command not found

Help! Ayuda! Aide-moi!! 😸

**Comment by bennypowers** [comment, marked as answer]
Please try with latest version

**Reply by bennypowers** [reply]
@eyevana did this solve your problem?

**Reply by eyevana** [reply]
@bennypowers looks like it installed this time. Thanks! 👍🏼 I'll continue working on integrating the new footer in the catalog. If anything else comes up, I'll let ya know.

**Reply by bennypowers** [reply]
Wonderful! thanks for confirming and do be in touch about RHDS in the future.
